### PR TITLE
lps-85543

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:help

**Comment by lawrence-lee** [comment]
ci:test:gradle-plugins

**Comment by lawrence-lee** [comment]
ci:reopen

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by lawrence-lee** [comment]
ci:test:gradle-plugins

File: test.properties [hunk 1]
```diff
@@ -648,8 +648,7 @@
         gradle-plugins-gulp,\
         gradle-plugins-js-module-config-generator,\
         gradle-plugins-js-transpiler,\
-        gradle-plugins-node,\
-        gradle-plugins-workspace
+        gradle-plugins-node
```
Comment (gamerson) [inline test.properties:651]: So this means that the "gradle-plugins" will run everything but the ones here that are excluded?

Comment (lawrence-lee) [reply test.properties:651]: yes. these were excluded from before. Andrea was getting daily emails about the ones failing.

**Comment by lawrence-lee** [comment]
ci:test:gradle-plugins

**Comment by lawrence-lee** [comment]
repushed here: https://github.com/gamerson/liferay-portal/pull/207

**Pull request closed without merging**